evalml/tests/component_tests/test_utils.py

```diff
@@ -149,7 +152,9 @@ class NonComponent:
 
 
 def test_scikit_learn_wrapper_invalid_problem_type():
-    evalml_pipeline = MulticlassClassificationPipeline([RandomForestClassifier])
+    evalml_pipeline = MulticlassClassificationPipeline(
+        [RandomForestClassifier],
+    )
```

Comment:
I might be in a weird state in VSCode with the new ruff linting, so idk if this and the above change are expected or indicative of my own weird environment. Doesn't seem like I have any linting issues in this PR so I'm fine leaving as is.